.jscs.json

```diff
@@ -15,28 +18,8 @@
         "common.blocks/ecma/__array/ecma__array.spec.js",
         "common.blocks/jquery/__event/_type/jquery__event_type_pointerclick.js",
         "common.blocks/jquery/__event/_type/jquery__event_type_pointer.js",
-        "common.blocks/jquery/__event/_type/jquery__event_type_pointerpressrelease.js"
+        "common.blocks/jquery/__event/_type/jquery__event_type_pointerpressrelease.js",
```

Comment:
Looks like this `jquery__event_type*` section is obsoleted. 

At present, only `jquery__event_type_pointerclick` is a vendor plugin and should not be checked. `jquery__event_type_pointer` was renamed to `jquery__event_type_pointernative`, so we can remove this line; as well ass `jquery__event_type_pointerpressrelease`. It was rewritten as a part of bem-core, so it now longer a vendor plugin.